server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/MinorCompactor.java

```diff
@@ -138,7 +138,7 @@ public CompactionStats call() {
           ProblemReports.getInstance(tabletServer).report(new ProblemReport(
               getExtent().getTableId(), ProblemType.FILE_WRITE, outputFileName, e));
           reportedProblem = true;
-        } catch (RuntimeException e) {
+        } catch (RuntimeException | NoClassDefFoundError e) {
```

Comment:
This has nothing to do with your change.  I was puzzled what the diff between this catch block and the prev one was looking at the code.  AFAICT the only diff is that one passes the exception to the log.warn and the other does not.

Reply:
That is curious and perhaps not the cleanest.  In the case of a NoClassDefFoundError I think I would want to see the exception in case the issue is more dire than a temporary VFS issue.